test/test_helper.rb

```diff
@@ -0,0 +1,18 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  fixtures :all     
+
+  Time.zone = 'Alaska'
+  Time.now
+  Date.today
+begin
+  
+rescue Exception
```

Comment:
__[staging][rails_best_practices]__
* not rescue Exception [:ref](http://rails-bestpractices.com/posts/2012/11/01/don-t-rescue-exception-rescue-standarderror/)

<!-- Comment by [SideCI](https://sideci.com) -->